Client/TeamTalkClassic/TeamTalkBase.h

```diff
@@ -203,17 +203,19 @@ enum StatusMode
     STATUSMODE_AWAY             = 0x00000001, 
     STATUSMODE_QUESTION         = 0x00000002, 
     STATUSMODE_MASK             = 0x000000FF,
+    STATUSMODE_MALE             = 0x00000003,

```

Comment:
This is a bitmask, so STATUSMODE_MALE will be the same as STATUSMODE_AWAY | STATUSMODE_QUESTION when it's 3. Change STATUSMODE_MALE to 0x00000000 (or same as STATUSMODE_AVAILABLE).

Start from bottom so STATUSMODE_NEUTRAL should be 0x00001000